docs/source/intervention_models/wasting_treatment/index.rst

```diff
@@ -298,30 +252,61 @@ Congratulate and counsel the mother on appropriate IYCF practices.
   The table below provides example entries for large scale food fortification with iron.
 
 .. list-table:: Affected Outcomes
-  :widths: 15 15 15 15
+  :widths: 15 15 15 30
   :header-rows: 1
 
   * - Outcome
     - Effect
     - Modeled?
     - Note (ex: is this relationship direct or mediated?)
-  * - Hemoglobin concentration
-    - Increases population mean
+  * - Median time-to-recovery
+    - Decreases 
     - Yes
-    - 
-  * - Malaria
-    - Increases incidence rate
+    - Direct relationship
+  * - Death rate
+    - Decreases death rate among the treated
     - No
-    - 
+    - Currently we do not have the relative risk of death of treated/untreated. We will capture some of the effect of tx on death by having the simulants recover faster, therefore leave the high risk state faster. However, by assigning utx simulants the mean death rate when they should have a higher death rate, we might overestimate the protective effect of tx on death.
+  * - Proportion recovered within programme treatment length
+    - Increases
+    - No (but we can)
+    - We assume that all treatment covered will recover within the time-to-recovery if they had not died. We can modify the treatment coverage by multiplying it by the 'cure rate' giving us the 'met-need' instead. 1 - 'met-need' will be the 'utx' population. 
+  * - Proportion defaulted during programme
+    - Decreases
+    - We can capture this by using 'met-need'
+    - This will decrease as 'met-need' increases.
+
```

Comment:
Maybe Abie and Nicole have diverging understanding of the treatment model and that is making it hard for us to come to consensus on the wasting model.  For discussion!

Reply:
we will get there!! :)